sql/src/test/java/org/apache/druid/sql/calcite/CalciteJoinQueryTest.java

```diff
@@ -723,7 +723,7 @@ public void testFilterAndGroupByLookupUsingJoinOperatorBackwards(Map<String, Obj
                                 ),
                                 "j0.",
                                 equalsCondition(makeColumnExpression("k"), makeColumnExpression("j0.dim2")),
-                                JoinType.RIGHT
+                                JoinType.INNER
```

Comment:
Same comment as before, we are converting a right join to an inner join here and that are changing the expected results